.github/workflows/tests.yml

```diff
@@ -32,9 +32,9 @@ jobs:
         run: $(go env GOPATH)/bin/golangci-lint run
 
   rpm_build:
-    name: "RPM build"
-    needs:
-      - lint
+    name: "📦 RPM"
+    # needs:
+    #   - lint
```

Comment:
Is this commented out by mistake?

Reply:
Yeah, I plan to just remove it so the RPM builds will finish sooner.